core-bundle/composer.json

```diff
@@ -102,7 +102,7 @@
         "symfony/event-dispatcher": "^5.4",
         "symfony/event-dispatcher-contracts": "^2.0 || ^3.0",
         "symfony/expression-language": "^5.4",
-        "symfony/filesystem": "^5.4",
```

Comment:
Actually, this change should be in the `manager-bundle`, not the `core-bundle`, as only the `manager-bundle` uses the `DotenvDumper`.

Reply:
Done in 0a329a3c8d1d86982315143502b6c875ed65c283